compiler/src/iree/compiler/Dialect/Stream/Transforms/test/encode_host_tensors_packing_i1.mlir

```diff
@@ -0,0 +1,20 @@
+// RUN: iree-opt --split-input-file --iree-stream-encode-host-tensors --verify-diagnostics --iree-experimental-packed-i1-storage %s | FileCheck %s
```

Comment:
We don't need the `--verify-diagnostics` flag because we're not testing the failures.